### ISSUE TITLE
Add global consumer group dashboard

### ISSUE THREAD

**Issue by Liu-XinYuan** [didi/Logi-KafkaManager#268]
For the scenario of Kafka cluster synchronization, we need to find a consumer group with a large lag. So global monitoring dashboard is very good and necessary

**Comment by ZQKC** [comment]
yes, a fantastic suggestion, in v3.0 will add this demand. and now, we have sink consumer lag to monitor system(default n9e), and you can see it in monitor system
